tests/functions.sh

```diff
@@ -212,27 +212,33 @@ messages {
   }
 }
 " > $f
-   
+    request $but_id $nic_id $f
+}
+
+function request {
```

Comment:
Create an intermediate function can be a good idea, but you need:
1: to explain why you do it
2: create it in another patch, because it's not related to renaming things